components/popover/style/index.less

```diff
@@ -15,6 +15,7 @@
   top: 0;
   left: 0;
   z-index: @zindex-popover;
+  max-width: 100%;
```

Comment:
I think this may break some existing behavior, as for our app, due to the addition of the max-width, our popover now inherit the width from ancestors instead of `none` (no size limit) previously, so we have to overwrite this back to `none` to fix that